wandb/wandb_run.py

```diff
@@ -98,6 +102,21 @@ def _init_jupyter_agent(self):
     def _stop_jupyter_agent(self):
         self._jupyter_agent.stop()
 
+    def configure(self, options):
```

Comment:
Could you call this something more specific than "configure"? Especially since it seems like this actually only does only thing and the terminology here doesn't match what it's called in RunManager.

Reply:
Fixed in feature/html